public/index.html

```diff
@@ -34,6 +34,70 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
   <title>Kickback</title>
+
+  <!-- 
+    Unlock snippet scripts modified from 
+    https://github.com/unlock-protocol/react-example/blob/master/public/index.html
+  -->
+  <script> (function (d, s) {
+    // if production, set network to Mainnet, else set to Rinkeby for paywall
+    const environment = "%NODE_ENV%";
+    const networkId = environment === "production" ? 1  : 4;
+
+    // if Rinkeby set, use Rinkeby paywall, else use Mainnet
+    let testnetUrl = networkId === 4 ? 'staging-' : '';
+    let js = d.createElement(s);
+    let sc = d.getElementsByTagName(s)[0];
+      js.src = `https://${testnetUrl}paywall.unlock-protocol.com/static/unlock.1.0.min.js`;
+      sc.parentNode.insertBefore(js, sc);
+    }(document, "script"));
+  </script>
+
+  <script>
+    // if production, set network to Mainnet, else set to Rinkeby for paywall
+    const environment = "%NODE_ENV%";
+    const networkId = environment === "production" ? 1  : 4;
+
+    // Kovan is included, but currently only Rinkeby and Mainnet is unsupported with paywall
+    // tokens entered first to last are displayed right to left in paywall modal
+    // NOTE: Replace Mainnet keys with keys Kickback owns
+    const locks = {
+      1: {
+        "enter-key-address-here": {
+          name: "Kickback Gold Supporter"
+        },
+        "enter-key-address-here": {
+          name: "Kickback Bronze Supporter"
+        }
+      },
+      4: {
+        "0xb452A1BfE64036718aCeFC8B5F718E0c9d233cB8": {
+          name: "Kickback Gold Supporter"
+        },
+        "0x70456ebD53E19eaE05932C482FeF1b24251ABb18": {
+          name: "Kickback Bronze Supporter"
+        }
+      },
+      42: {
+        "0x2eD8aD6AA973BEA5D60FA70Eead10F113fb4Dc99": {
+          name: "Kickback Gold Supporter"
+        },
+        "0xe61607898287220586abE01b536714d8429f675C": {
+          name: "Kickback Bronze Supporter"
+        }
+      }
+    }[networkId]
+
+    // unlock config must use var
+    var unlockProtocolConfig = {
+      locks,
+      icon: 'https://unlock-protocol.com/static/images/svg/unlock-word-mark.svg',
```

Comment:
This could be changed to a Kickback logo ;)

Reply:
Good idea. Changed in a60f5f7. It looks like only public urls are compatible with Unlock, so I used https://kickback.events/card.png.